src/camera-controls.js

```diff
@@ -882,6 +902,45 @@ export default class CameraControls extends EventDispatcher {
 
 	}
 
+	_encloseToBoundary( position, offset, friction ) {
```

Comment:
Do we need the third argument? we could use this. `boundaryFriction` in the method.
```suggestion
	_encloseToBoundary( position, offset ) {
```